### PR TITLE
User Guide not updated with the new commands and prefixes

### LINKED ISSUE #69: UG is not updated with the new commands and new prefixes

### PR DESCRIPTION
**Pull request by AD-NAP**
User Guide still contain the old commands and discarded commands such as addinfo command. Table of content is also not updated.

Let's 
* update user guide with the new commands for adding and flagging
clients. 
* update table of content of UG

Note: UG Section on deleting and editing by name will be done by Vish.

This should close #69. Nice.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -78,48 +86,49 @@ Shows a message explaining how to access the help page.
...
 Edits an existing person in the HustleBook.
 
-* Using **Index**:
+* Using **Name**:
 
-  Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`
+  Format: `edit NAME [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [i/INFO] [d/DATE] [t/TAG]…​`
 
-  * Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index **must be a positive integer** 1, 2, 3, …​
+  * Edits the person named `NAME`. The index refers to the index number shown in the displayed person list. The index **must be a positive integer** 1, 2, 3, …​
```
Comment (Vishwanth2210) [inline docs/UserGuide.md:131]: It would be better if the second statement can be deleted as there is no more index that can be used for this command.

**Review by Vishwanth2210 — CHANGES_REQUESTED** [review]
Looks good except for that one small statement. Other than that, it looks good to be merged.

File: docs/UserGuide.md (hunk 1 above)
Comment (AD-NAP) [reply docs/UserGuide.md]: Okay got it 😄

**Review by Vishwanth2210 — APPROVED** [review]
Looks good

**Pull request merged**